README.md

```diff
@@ -0,0 +1,406 @@
+```
+
+### Downloads for a single YYYY-MM month
+
+```console
+    $ pypinfo --month 2018-04 setuptools
+    Served from cache: False
+    Data processed: 571.37 MiB
+    Data billed: 572.00 MiB
+    Estimated cost: $0.01
+
+    | download_count |
+    | -------------- |
+    |      8,972,826 |
+```
+
+### Percentage of Python 3 downloads of the top 100 projects in the past year
+
+Let's use ``--test`` to only see the query instead of sending it.
+
+```console
+    $ pypinfo --test --days 365 --limit 100 "" project percent3
+    SELECT
+        file.project as project,
+        ROUND(100 * SUM(CASE WHEN REGEXP_EXTRACT(details.python, r"^([^\.]+)") = "3" THEN 1 ELSE 0 END) / COUNT(*), 1) as percent_3,
+        COUNT(*) as download_count,
+    FROM `bigquery-public-data.pypi.file_downloads`
+    WHERE timestamp BETWEEN TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -366 DAY) AND TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -1 DAY)
+        AND details.installer.name = "pip"
+    GROUP BY
+        project
+    ORDER BY
+        download_count DESC
+    LIMIT 100
+```
+
+### Downloads for a given version
+
+pypinfo supports `PEP 440 version matching <https://peps.python.org/pep-0440/#version-matching>`_.
+
+We can use it to query stats on a given major version.
+
+```console
+    $ pypinfo -pc 'pip==21.*' pyversion version
+    Served from cache: False
+    Data processed: 34.45 MiB
+    Data billed: 35.00 MiB
+    Estimated cost: $0.01
+
+    | python_version | version | percent | download_count |
+    | -------------- | ------- | ------- | -------------- |
+    | 3.6            | 21.3.1  |  78.74% |         10,430 |
+    | 3.8            | 21.3.1  |   7.81% |          1,034 |
+    | 3.7            | 21.2.1  |   3.59% |            476 |
+    | 3.7            | 21.3.1  |   2.60% |            345 |
+    | 3.7            | 21.0.1  |   2.25% |            298 |
+    | 3.8            | 21.0.1  |   1.58% |            209 |
+    | 3.8            | 21.2.1  |   1.42% |            188 |
+    | 3.7            | 21.1.2  |   0.81% |            107 |
+    | 3.9            | 21.3.1  |   0.69% |             92 |
+    | 3.8            | 21.1.1  |   0.51% |             67 |
+    | Total          |         |         |         13,246 |
+```
+
+We can also use it to query stats on an exact version:
+
+```console
+    $ pypinfo -pc 'numpy==1.23rc3' pyversion version
+    Served from cache: False
+    Data processed: 34.01 MiB
+    Data billed: 35.00 MiB
+    Estimated cost: $0.01
+
+    | python_version | version   | percent | download_count |
+    | -------------- | --------- | ------- | -------------- |
+    | 3.9            | 1.23.0rc3 |  63.33% |             38 |
+    | 3.8            | 1.23.0rc3 |  28.33% |             17 |
+    | 3.10           | 1.23.0rc3 |   8.33% |              5 |
+    | Total          |           |         |             60 |
+```
+
+</details>
+
+
+## Installation
+
+pypinfo is distributed on **PyPI** as a universal wheel and is available on
```

Comment:
I was thinking wrap a single `<details>` around the whole installation bit, rather than each image?

So it would look more like the Usage part:

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/1324225/224503673-38374afe-232e-48de-8dec-2d50ae0c89e8.png">

Reply:
I can wrap the images AND the installation section (Related pun: https://www.youtube.com/watch?v=vgk-lA12FBk). A bold-face remark in the introduction will indicate the helpful images. What do you think?

Reply:
@hugovk I edited like you please https://github.com/brunolnetto/pypinfo